### PR TITLE
Added trace, debug and fatal logger levels.

### PR DESCRIPTION
**Pull request by newboadki**

### CODE REVIEW

File: Harmony/Classes/Alamofire/URLLoggerRequestAdapter.swift [hunk 1]
```diff
@@ -31,7 +31,7 @@ public class URLLoggerRequestAdapter: RequestAdapter {
     
     public func adapt(_ urlRequest: URLRequest) throws -> URLRequest {
         let curl = urlRequest.curl(pretty: true)
-        logger.print(tag: tag, "\(curl)")
```
Comment (newboadki) [inline Harmony/Classes/Alamofire/URLLoggerRequestAdapter.swift:34]: Note that in order to match the changes in Asier's pull request, I deleted a method, which will potentially require clients of this library to be updated.

Comment (joselufo) [reply Harmony/Classes/Alamofire/URLLoggerRequestAdapter.swift:34]: It's ok, we will update it when the project doesn't compile we can do a _find & replace_

Thanks.

**Review by joselufo — APPROVED** [review]
All ok from my side.

**Comment by newboadki** [comment]
Are there more people interested in reviewing this? or should this get merged and the related issue closed?

**Comment by joselufo** [comment]
It's been here the pull-request for at least a week so it's ok to merge if no more people want to check it. 

So merge it.

**Pull request merged**